tests/test_units.py

```diff
@@ -0,0 +1,108 @@
+from rdflib import Namespace
+import re
+import brickschema
+from collections import defaultdict
+import sys
+
+sys.path.append("..")
+from bricksrc.namespaces import A, BRICK, TAG, QUDT  # noqa: E402
+
+BLDG = Namespace("https://brickschema.org/schema/ExampleBuilding#")
+
+
+def test_quantity_has_one_quantitykind():
+    """
+    In the current implementation, using owl:equivalentClass to align Quantity
+    with QUDT QuantityKinds, we need to make sure that through RDFS subclassing
+    that a Brick Quantity does not end up with more than 1 QuantityKind
+    """
+    g = brickschema.graph.Graph()
+    g.load_file("Brick.ttl")
+    g.g.bind("qudt", QUDT)
+    g = brickschema.inference.OWLRLInferenceSession(load_brick=False).expand(g)
+    quantity_qk = g.query(
```

Comment:
Can this cover Brick Quantities not associated with any QUDT QuantityKinds?

Reply:
No. This is only testing that each Brick Quantity is only associated with a single QUDT QuantityKind.